lib/ansible/module_utils/crypto.py

```diff
@@ -18,6 +18,10 @@
 
 try:
     from OpenSSL import crypto
+    from cryptography import x509
```

Comment:
You should move that into a different `try/except` block, otherwise not having PyOpenSSL installed will prevent `cryptography` from being importet.